ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -3,6 +3,8 @@
  * @author shuanglihtk@google.com (Shuang Li)
  */
 
+const TWOPOINTS = 6;
```

Comment:
nit: 

// the length of two 3D coordinates (x, y, z) in the coordinate lookup float array
const TWO_POINTS = 6;